user/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+import json, bcrypt, jwt
 
-# Create your views here.
+from django.shortcuts       import render
+from django.views           import View
+from django.http            import JsonResponse, HttpResponse
+
+from user.models            import User
+from my_settings            import SECRET_KEY, ALGORITHM
+
+class SignInView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        try:
+            user_name   = data['user_name']
+            password    = data['password']
+            
+            if not User.objects.filter(user_name=user_name).exists():
+                return JsonResponse({'message':'INVALID_USER'}, status=401)
+            
+            user = User.objects.get(user_name=user_name)
+
+            password_validation = bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8'))
+
+            if password_validation:
+                acces_token = jwt.encode({'user_id':user.id}, SECRET_KEY, algorithm=ALGORITHM)
+
+                return JsonResponse({'message':'SUCESS', "token":acces_token}, status=200)
```

Comment:
서버에서 발급해주는 token이 어떤 token인지 명확하게 만들어주세요.
나중에는 refresh_token 등 다양한 토큰을 서버에서 관리해야되는데, `token`은 어떤 토큰인지 의미가 분명하지 않습니다.
```suggestion
                return JsonResponse({'message':'SUCESS', "access_token":acces_token}, status=200)
```

Reply:
수정했습니다!